core/sail/memory/src/test/resources/benchmarkFiles/particularly-large-join-surface.qr

```diff
@@ -0,0 +1,17 @@
+PREFIX ex: <http://example.com/ns#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX sh: <http://www.w3.org/ns/shacl#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+PREFIX dcat: <http://www.w3.org/ns/dcat#>
+PREFIX dc: <http://purl.org/dc/terms/>
+PREFIX skos:  <http://www.w3.org/2004/02/skos/core#>
+PREFIX foaf:  <http://xmlns.com/foaf/0.1/>
+PREFIX dct: <http://purl.org/dc/terms/>
+
+SELECT distinct ?type where {
+ 	?a (<:> | !<:>) / (<:> | !<:>) / (<:> | !<:>)  / a ?type.
+   	?a (<:> | !<:>) / (<:> | !<:>) / a ?type.
```

Comment:
This look like ASCII-art - honestly SPARQL is right up there with Perl in its ability to obfuscate.